cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -251,7 +251,7 @@ def continue_cherry_pick(self):
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
 @click.command(context_settings=CONTEXT_SETTINGS)
-@click.version_option(version="0.0.2.post2")
+@click.version_option(version="0.1.0.dev1")
```

Comment:
Why not `from . import __version__`?  Or the other way around (define it here, import it in `__init__.py`), if flit supports that?

I'm not really clear on why exactly `__init__.py` is necessary at all :)